File: 0283-move-zeroes/0283-move-zeroes.c
void moveZeroes(int* n, int ns) {
    int c=0;
    for(int i=0;i<ns;i++)
    {
        if(n[i]!=0)
        {
            int t = n[i];
            n[i]=n[c];
            n[c]=t;
            c++;
        }
       
    }

}

File: 0283-move-zeroes/0283-move-zeroes.java
class Solution {
    public void moveZeroes(int[] nums) {
        int no = 0;
        for(int i=0;i<nums.length;i++)
        {
            if(nums[i]!=0)
            {
                int t = nums[no];
                nums[no]=nums[i];
                nums[i]=t;

                no++;
            }
        }
        
    }
}